.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216616995-843c8e4b-4fcf-4329-8fa2-f1e49478dee3.png)

**[Optional]**
- [ ] Your commit messages are not meaningful :fearful: Kindly make them meaningful to make your pull request professional :+1: